rae_msgs/msg/ColorPeriod.msg

```diff
@@ -0,0 +1,5 @@
+float32 r
```

Comment:
You can use ColorRGBA message instead

Reply:
Are you sure? ColorRGBA doesnt have additional variable I use to control intensity of the light.

Reply:
I meant using ColorRGBA as a field here, so the message would be as follows:
```
std_msgs/ColorRGBA color
float32 period
```

Reply:
Sure - I can do it that way too. Thought it would be bit annoying to fill 2 different arrays (one for color and one for period since every LED can have different period) for users, but maybe this way it can be more intuitive for users.